controllers/provider-openstack/charts/internal/cloud-provider-config/templates/cloud-provider-config-intree.yaml

```
@@ -0,0 +1,30 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ .Values.inTreeCloudProviderConfigName }}
```

What's the need to have this as value? 
```suggestion
  name: cloud-provider-config-kube-controller-manager
```